integrationExamples/gpt/top-level-paapi/decisionLogic.js

```diff
@@ -0,0 +1,87 @@
+// taken from https://privacysandboxdemos-seller.casalemedia.com/ssp/decision-logic.js
+
+function log(label, o) {
+  console.debug(label, JSON.stringify(o, ' ', ' '));
+}
+
+function scoreAd(
+  adMetadata,
+  bid,
+  auctionConfig,
+  trustedScoringSignals,
+  browserSignals,
+  directFromSellerSignals,
+) {
+  var score = bid+20;
```

Comment:
reference comparison to floor (or top prebid bid)


  var sellerSignals = auctionConfig.sellerSignals;
  var floor = sellerSignals.floor || 0;
  var modifiedBid = bid * 0.95;

  // temporary reporting: https://github.com/WICG/turtledove/blob/main/Proposed_First_FLEDGE_OT_Details.md#reporting
  const reportingBrowserSignals = {...browserSignals, bid, modifiedBid}
  const macros = '&winningBid=${winningBid}&winningBidCurrency=${winningBidCurrency}&topLevelWinningBid=${topLevelWinningBid}&topLevelWinningBidCurrency=${topLevelWinningBidCurrency}';
  globalThis.forDebuggingOnly?.reportAdAuctionLoss(
    buildReportURL('bid-loss', auctionConfig, reportingBrowserSignals) + macros);
  globalThis.forDebuggingOnly?.reportAdAuctionWin(
    buildReportURL('bid-win',  auctionConfig, reportingBrowserSignals) + macros);

  return {
    ad: null,
    desirability: (modifiedBid > floor && (browserSignals.bidCurrency == '???' || auctionConfig.sellerCurrency == browserSignals.bidCurrency)) ? modifiedBid : 0,
    allowComponentAuction: true,
    bid: modifiedBid
  };
}

Reply:
citing my source ```https://privacysandbox.openx.net/fledge/decision-logic-component.js````